### ISSUE TITLE
lpSolve update from CRAN

### ISSUE THREAD

**Issue by gaborcsardi** [gaborcsardi/lpSolve#19]
```
Version: 5.6.19
Check: whether package can be installed
Result: WARN 
  Found the following significant warnings:
    hbio.c:1629:11: warning: format string is not a string literal (potentially insecure) [-Wformat-security]
  See ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/lpSolve.Rcheck/00install.out’ for details.
  * used C compiler: ‘Debian clang version 17.0.5 (1)’
Flavor: [r-devel-linux-x86_64-debian-clang](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-clang/lpSolve-00check.html)
```

**Comment by gaborcsardi** [comment]
Issue fixed in 2daf437dfcb0f36d9a2e32d5098dee778f1f1f68

**Comment by gaborcsardi** [comment]
Done in #20.
